### DISCUSSION TITLE
Expand/collapse levels in left menu

### DISCUSSION THREAD

**Discussion by denisemcg** [GitbookIO/community#708] [Feature Requests]
My left menu is very long, and users must scroll a lot to get to the pages at the bottom. There's no need for the full menu to be expanded. I'd like to be able to collapse the menu at the top-level items.
Can you please look into implementing expand/collapse functionality, as it's not a great experience for my users at the moment. 
![image](https://github.com/user-attachments/assets/1ce8bad2-09f9-46d3-b2c9-53a1ea1d7e89)

There are other discussions about the left menu so I think an enhancement in this area should be investigated.
https://github.com/orgs/GitbookIO/discussions/642   ***especially this one***
https://github.com/orgs/GitbookIO/discussions/545 
https://github.com/orgs/GitbookIO/discussions/683
https://github.com/orgs/GitbookIO/discussions/441

**Comment by addisonschultz** [comment]
Hey @denisemcg - I see you're using Groups for this - which aren't able to be collapsed in the published view. Could you try using a page (and nesting the inner pages inside)? You can also add custom icons or emojis to pages - making it easy to read at a glance!

**Reply by denisemcg** [reply]
As simple as that! Thanks very much. It's looking much better now.

**Reply by denisemcg** [reply]
@addisonschultz can I really add 'custom' emojis? I mean, upload and use our branded icons as emojis. I don't want to use any of the inbuilt icons/emojis.

**Reply by addisonschultz** [reply]
Hey @denisemcg - right not it's a collection of icons from https://fontawesome.com/icons

We're still looking into how we can best support custom icons, but right now they are unfortunately not supported

**Reply by aikoii** [reply]
@denisemcg Hey! Good news - page groups are now collapsible 😊

https://github.com/user-attachments/assets/f57997f8-7405-443e-b5df-7f870a8674c8
